### ISSUE TITLE
Compute address usage in wallet

### ISSUE THREAD

**Issue by mariocao** [witnet/witnet-rust#1192]
While indexing wallet transactions, the wallet server should also compute if wallet PKHs are being used in transaction while keeping the following information:

 - amount: received amount to that address
 - first_payment_date: timestamp from the first received movement
 - last_payment_date: timestamp from the last received movement
 - payments: number of payments received

Note: the `amount` does not reflect if amount received has been spent after the payment or not.

**Comment by mariocao** [comment]
@Tommytrg proposed here https://github.com/witnet/witnet-rust/issues/1206 to extend the address schema with the following fields:

```
type Address = {
 // Number of payments receive in the address
 payments: Number
 // When the address received the first payment
 first_payment_date: Timestamp,
 // When the address received the last payment
 last_payment_date: Timestamp
 // Wits received in the address
 amount: Number
}
```

**Comment by tmpolaczyk** [comment]
Implemented in #1207
